{"name": "Проверка возврата ошибки при указании неправильных логина или пароля", "status": "passed", "start": 1728215415669, "stop": 1728215415775, "uuid": "2b10be10-5591-4cc7-b8cc-1f84a23ec728", "historyId": "705d31d231a05e77e20f16a9e5201fcb", "testCaseId": "705d31d231a05e77e20f16a9e5201fcb", "fullName": "tests.test_courier_log_in.TestCourierLogIn#test_error_when_login_or_password_incorrect", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_log_in"}, {"name": "subSuite", "value": "TestCourierLogIn"}, {"name": "host", "value": "DESKTOP-ABCQNV2"}, {"name": "thread", "value": "10120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_log_in"}]}